src/main/java/frc/robot/commands/intake/Actuator.java

```diff
@@ -16,4 +16,8 @@ public Actuator(Intake intake) {
 	public void execute(){
 		this.intake.toggle();
 	}
+	public void end(boolean interrupted) {
+		this.intake.stop();
```

Comment:
teoricamente n precisaria do this.intake.stop() já que no actuator os motores não são acionados